setup.py

```diff
@@ -1,21 +1,42 @@
-#!/usr/bin/env python3
+ #!/usr/bin/env python3
 
 from setuptools import setup
 
+import os.path
+import sys
+import warnings 
+
+# Changed paths to remove absolutes
+
+files_spec = [
+        ('share/wikicurses', ['conf/wikicurses.conf.dist']),
+        ('share/zsh/site-functions', ['zsh/_wikicurses']),
+	('share/man/man1', ['docs/wikicurses.1']),
```

Comment:
Hm. Unfortunately `man` doesn't seem to check `~/.local/share/main` by default, at least on Arch Linux. But I presume there's no better solution for user-local installs...